plugin/auth/src/main/java/com/alibaba/nacos/plugin/auth/constant/Constants.java

```diff
@@ -36,6 +36,16 @@ public static class Auth {
         public static final String NACOS_CORE_AUTH_SERVER_IDENTITY_VALUE = "nacos.core.auth.server.identity.value";
         
         public static final String NACOS_CORE_AUTH_ENABLE_USER_AGENT_AUTH_WHITE = "nacos.core.auth.enable.userAgentAuthWhite";
+    
+        public static final String NACOS_CORE_AUTH_LDAP_URL = "nacos.core.auth.ldap.url";
```

Comment:
ladp plugin constants should be in plugin-impl module

Reply:
I will modify it next week.